gdal/frmts/derived/deriveddataset.cpp

```diff
@@ -37,8 +37,10 @@ class DerivedDataset : public VRTDataset
     public:
         DerivedDataset(int nXSize, int nYSize);
         ~DerivedDataset();
-
+  
+        static int Identify( GDALOpenInfo * );
```

Comment:
Oh you're right. I didn't actually check the existing code, since I didn't see how that would have compiled. So I guess that currently it is the Identify static method of the VRTDataset that is used!